### DISCUSSION TITLE
Разнообразие в оружии для сержанта

### DISCUSSION THREAD

**Discussion by Kyper1** [octothorp-team/public-dobrograd#624] [Предложения]
Все легко и просто, добавить сержанту еще оружие. а если по конкретней, то m4a1. 
Но носить с собой он может только m4a1 либо m3. Выбирать он сможет в экипировочной или чередовать их, используя багажник своего служебного автомобиля. 

Плюсы: 

- Если произошла ситуация, где необходимо применить табельное и к примеру действия происходят в большом переулке, то сержанту не придется ходить, опасаясь всего с глоком, а сможет, как уверенный суровый мужчина проверять углы с автоматической винтовкой. 
- Количество людей, хотящих приобрести плюшку добродей из-за ее профессий, включая сержанта, возрастет минимум в 1.5

Минусы: 

- Краймерам будет нелегко 

Причины добавить:

- Человек платит 250Р в месяц, ну дайте ему повеселиться.

**Comment by rasusibtisam** [comment]
Полиция и так сильна. У копов есть броня, у сержантов и броня, и дробовик, который сносит прилично урона, также и Dpd с автоматическими винтовками и возможностью экипировать SWAT снаряжение. Усилять и так сильную фракцию будет дисбалансно

**Reply by 1BillBlack** [reply]
> Полиция и так сильна. У копов есть броня, у сержантов и броня, и дробовик, который сносит прилично урона, также и Dpd с автоматическими винтовками и возможностью экипировать SWAT снаряжение. Усилять и так сильную фракцию будет дисбалансно

На самом деле краймеры, кладущие весь ПД действительно показывают дисбаланс. Если гибнет столько офицеров, ну наверное, что-то не так. Даже беря в расчет то, что это игра, но делая упор на реальность: ни в одном штате США не гибнет столько полиции на одной ситуации. Даже за год столько не гибнет, сколько на ДБГ.

**Reply by rasusibtisam** [reply]
Я думаю причиной гибели множества офицеров является отсутствие инфы о том кто противник, т.к. краймер любой  -гражданский, который может войти в бой в самый неожиданный момент. Я смотрел видос "Бойня на рузвельт"(https://www.youtube.com/watch?v=RgMyH9mUjuM), где было около 14 трупов, т.к. краймеров было и копы не могли на каждого титизена наставить ствол. Так что тут кол-во трупов зависит от кол-ва краймеров и вооружения краймеров(а оружие у нас очень дорогое, так что это очень даже правильно в плане баланса).

**Comment by Ggman77** [comment]
думаю это будет логично, я за
